tests/performance_tests/performance_test_basic.py

```diff
@@ -21,26 +21,144 @@
 relaunchTimeout = 30
 emptyBlockGoal = 5
 
-def waitForEmptyBlocks(node1):
+class blockData():
```

Comment:
This should be a Python 3 [data class](https://docs.python.org/3/library/dataclasses.html).